core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -366,6 +371,17 @@ contract PricelessPositionManager is FeePayer {
         emit SettleExpiredPosition(msg.sender, totalRedeemableCollateral.rawValue, tokensToRedeem.rawValue);
     }
 
+    /**
+     * @notice If token holders or sponsors do not redeem their positions for underlying collateral within
+     * a pre-defined `siphonDelay`, all underlying collateral is sent to the Store. This delay is set to
+     * be sufficiently long such that token holders have a reasonable period of time to withdrawal (>6 months).
+     */
+    function siphonContractCollateral() external {
+        require(getCurrentTime() > expirationTimestamp + siphonDelay);
+        uint contractCollateralBalance = collateralCurrency.balanceOf(address(this));
+        collateralCurrency.safeTransfer(_getStoreAddress(), contractCollateralBalance);
```

Comment:
Can you write up (maybe in the PR description) what happens to all the contract functions after this is called? Are there any unintended consequences wrt contract bookkeeping or user interaction?

Reply:
done